apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/connect-term-to-video.e2e.cy.ts

```diff
@@ -0,0 +1,130 @@
+describe(`term detail flow`, () => {
+    describe(`connect resources with note`, () => {
+        const createConnectionCommandLabel = `Create Connection with Note`;
+
+        const newNoteText = 'This is an interesting note.';
+
+        beforeEach(() => {
+            cy.visit(`Resoruces/Terms`);
```

Comment:
Does this work?  (sp) Resoruces

Reply:
Ah long story as to why it works. I'll fix it.

Reply:
fixed